x-pack/test/api_integration/apis/maps/get_tile.js

```diff
@@ -42,6 +44,29 @@ export default function ({ getService }) {
         ['machine.os.raw']: 'ios',
       });
       expect(feature.loadGeometry()).to.eql([[{ x: 44, y: 2382 }]]);
+
+      // Metadata feature
+      const metadataFeature = layer.feature(2);
+      expect(metadataFeature.type).to.be(3);
+      expect(metadataFeature.extent).to.be(4096);
+      expect(metadataFeature.id).to.be(undefined);
+      expect(metadataFeature.properties).to.eql({
+        __kbn_feature_count__: 2,
+        __kbn_is_tile_complete__: true,
+        __kbn_metadata_feature__: true,
+        __kbn_vector_shape_type_counts__: '{"POINT":2,"LINE":0,"POLYGON":0}',
+        fieldMeta:
+          '{"machine.os.raw":{"categories":{"categories":[{"key":"ios","count":1},{"count":1}]}},"bytes":{"range":{"min":9252,"max":9583,"delta":331},"categories":{"categories":[{"key":9252,"count":1},{"key":9583,"count":1}]}},"_index":{"categories":{"categories":[{"key":"logstash-2015.09.20","count":2}]}},"_id":{"categories":{"categories":[{"key":"AU_x3_BsGFA8no6Qjjug","count":1},{"key":"AU_x3_g3GFA8no6QjkGu","count":1}]}},"__kbn__feature_id__":{"categories":{"categories":[{"key":"logstash-2015.09.20:AU_x3_BsGFA8no6Qjjug:0","count":1},{"key":"logstash-2015.09.20:AU_x3_g3GFA8no6QjkGu:0","count":1}]}}}',
```

Comment:
You should exclude keys _index, _id, and __kbn__feature_id__ keys when generating the list. These are not selectable fields for styling and will never be needed in client.